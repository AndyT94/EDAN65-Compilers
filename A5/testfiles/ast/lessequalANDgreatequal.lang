int a() {
	if(a<=5) {
		a=1;
	}
	
	if(a>=5) {
		a=5;
	}
}