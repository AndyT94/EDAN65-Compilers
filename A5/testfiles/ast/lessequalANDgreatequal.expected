Program
  List
    Function
      IntType
      Id_decl ID="a"
      Params
        List
      Block
        List
          If
            LessEqual
              Id_use ID="a"
              Numeral NUMERAL="5"
            Block
              List
                Assign
                  Id_use ID="a"
                  Numeral NUMERAL="1"
            Opt
          If
            GreatEqual
              Id_use ID="a"
              Numeral NUMERAL="5"
            Block
              List
                Assign
                  Id_use ID="a"
                  Numeral NUMERAL="5"
            Opt
